dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -2587,6 +2587,10 @@ RtpsUdpDataLink::check_heartbeats()
 void
 RtpsUdpDataLink::send_relay_beacon()
 {
+  // Use a pad submessage with for the beacon
+  static const ::CORBA::Octet beacon_msg[] = { 1, 0, 0, 0 };
```

Comment:
This could use the `PAD` constant and also if we move it to a common header like `BaseMessageTypes.h` it can be reused by the relay code instead of being repeated there.